services/apps/alcs/src/providers/typeorm/migrations/1704396938830-refresh_views_for_hide_from_public.ts

```diff
@@ -0,0 +1,165 @@
+import { MigrationInterface, QueryRunner } from 'typeorm';
+
+export class RefreshViewsForHideFromPublic1704396938830
+  implements MigrationInterface
+{
+  name = 'RefreshViewsForHideFromPublic1704396938830';
+
```

Comment:
What is going on? 😵

Reply:
What do you mean? The name of the migration is RefreshViewsForHideFromPublic and the number after is the timestamp.

Reply:
I mean how come we need to do a big refresh? Does it happen periodically with the db?

Reply:
sorry, maybe I should use the word re-create, since i changed the query I need to drop and re-create the view which I called refreshing.